.github/workflows/lint-test.yml

```
@@ -121,13 +121,6 @@ jobs:
       - name: Run tests and generate coverage report
         run: pytest -n auto --cov --disable-warnings -q
```

```suggestion
        run: pytest -n auto --disable-warnings -q
```
Shouldn't the coverage report not be generated now?